docs/testbenches/ip_based/util_pack/index.rst

```diff
@@ -0,0 +1,277 @@
+.. _util_pack:
+
+Util Pack
+================================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The purpose of this testbench is to validate the serial interface functionality
+of the :git-hdl:`library/util_pack/util_cpack2 <library/util_pack/util_cpack2>`
+and :git-hdl:`library/util_pack/util_upack2 <library/util_pack/util_upack2>` IP
+cores.
+
+The entire HDL documentation can be found here
+:external+hdl:ref:`here <util_cpack2>` and here
```

Comment:
Remove "here" before the refs, because now it renders like this: 

The entire HDL documentation can be found here here and here here.

Suggestion:

The entire HDL documentation can be found :external+hdl:ref:`here <util_cpack2>` and :external+hdl:ref:`here <util_upack2>`.